graylog2-server/src/main/java/org/graylog/enterprise/EnterpriseLicenseInfo.java

```diff
@@ -14,14 +14,14 @@
  * along with this program. If not, see
  * <http://www.mongodb.com/licensing/server-side-public-license>.
  */
-package org.graylog.freeenterprise;
+package org.graylog.enterprise;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.auto.value.AutoValue;
 
 @AutoValue
-public abstract class FreeLicenseInfo {
+public abstract class EnterpriseLicenseInfo {
     public enum Status {
```

Comment:
Can we remove `STAGED`?

Reply:
Good catch, thank you! :+1: I removed `STAGED`.